core/src/main/java/org/bitcoinj/store/SPVBlockStore.java

```diff
@@ -267,6 +267,7 @@ public void close() throws BlockStoreException {
                 WindowsMMapHack.forceRelease(buffer);
             }
             buffer = null;  // Allow it to be GCd and the underlying file mapping to go away.
+            fileLock.release();
```

Comment:
Closing the file, closes the channel, which releases the fileLock as per java.nio.channels.FileLock's JavaDoc. This line of code is obsolete.